lib/Server.js

```diff
@@ -270,8 +270,10 @@ class Server {
     const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
     const hot = this.hot;
     const publicPath = hot ? (webpackURL + '/') : null;
+    const platform = this.entry.substring(this.entry.search(/\./) + 1);
```

Comment:
Right now `Server.js#constructor` defaults `this.entry` to `'index.ios'` if options are not passed from CLI. Certainly the purpose of the string parsing could be clearer here, as this will probably choke if an entry point or value is passed which doesn't contain the expected format `[platform].ios`.